portfolio.css

```diff
@@ -0,0 +1,574 @@
+ body{ 
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box; 
+  font-family: "SFproText", sans-serif;
+  background: #313131;
+  color: #F5F1F1;
+}
+
+
+.nav-bar {
+  box-sizing: border-box; 
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px 32px;
+  background: #1D1C1C;
+}
+.nav-bar a {
+  color: #444444;
+  text-decoration: none;
+}
+
+.nav-bar a:hover {
+  color: #267df4;
+}
+
+.nav-bar a:active {
+  color: #267df4;
+}
+
+.nav-bar a:focus {
+  color: #267df4;
+}
+
+
+.logo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.circle {
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  background-color: var(--portfolio-placeholder, #D9D9D9);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 10px;
+}
+
+.initial {
+  font-size: 48px;
+  color: var(--portfolio-text, #090808);
+}

```

Comment:
It would be good to have more descriptive classes. If I just look at the CSS file and see "circle" or "initial", I have no idea what in the page it could be and have to go and search for those classes in HTML. It is the best practice to have class names that provide more information at first glance, at the same time still keeping them simple. 
`circle` could be `nav-logo` or `nav-round-logo`
`initial` could be `nav-logo-letter`
This applies to more of your classes, I just took a few as examples 🙏🏽